contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/BasicFetchStrategy.java

```diff
@@ -89,82 +84,86 @@ private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType c
         OutputStream output = null;
         long startTimeMS = System.currentTimeMillis();
 
-        for (int attempt = 0; attempt < fetcher.getMaxAttempts(); attempt++) {
```

Comment:
I believe your settings remove the space between the for, while and if with single space. This introduced lots of change. I generally prefer it with the spaces between them.